.changeset/config.json

```diff
@@ -2,22 +2,30 @@
   "$schema": "https://unpkg.com/@changesets/config@2.2.0/schema.json",
   "changelog": "@changesets/cli/changelog",
   "commit": false,
-  "fixed": [],
-  "linked": [[
+  "//": "CAUTION: When updating the fields below, you should also update README accordingly.",
+  "fixed": [[
+    "@logto/cli",
+    "@logto/create"
+  ], [
     "@logto/console",
     "@logto/core",
-    "@logto/demo-app",
     "@logto/integration-tests",
+    "@logto/ui"
+  ]],
+  "//": "Ignore other release group members, only keep the major one.",
+  "ignore": [
+    "@logto/create",
+    "@logto/console",
+    "@logto/integration-tests",
+    "@logto/ui"
+  ],
```

Comment:
I don't quite get how this "ignore" field works. Can you point me to the documentation?

Reply:
https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#ignore-array-of-packages